src/pharmpy/modeling/odes.py

```diff
@@ -1505,7 +1505,7 @@ def has_first_order_absorption(model: Model):
 
     """
 
-    odes = model.statements.ode_system
+    odes = get_and_check_odes(model)
     if odes is None:
```

Comment:
Now this is not needed. `get_and_check_odes` guarantees the return of a `CompartmentalSystem` (else it raises)

Reply:
Oh sorry, I'll fix it right away!